workspaces/src/network/config.rs

```diff
@@ -37,20 +37,18 @@ fn overwrite(home_dir: &Path, value: Value) -> Result<()> {
     Ok(())
 }
 
-/// Limit how much nearcore/sandbox can receive per payload. The default set by nearcore is not
-/// enough for certain sandbox operations like patching contract state in the case of contracts
-/// larger than 10mb.
-fn max_sandbox_json_payload_size() -> Result<u64> {
-    let max_files = match std::env::var("NEAR_SANDBOX_MAX_PAYLOAD_SIZE") {
+/// Parse an environment variable or return a default value.
+fn parse_env_or<T>(env_var: &str, default: T) -> Result<T>
```

Comment:
A real nit, so feel free to ignore, but shouldn't this be:
```suggestion
fn parse_env<T>(env_var: &str, default: T) -> Option<Result<T>>
```
Then called as:
```rust
parse_env_or("NEAR_SANDBOX_MAX_FILES").unwrap_or(3000)
```
It just feels a bit more general and standard rust

Reply:
Or, alternatively, consider returning a `Result<Option<T>>`

so instead of

```rust
parse_env("NEAR_SANDBOX_MAX_FILES").unwrap_or(Ok(3000))?;
```

it'll be

```rust
parse_env("NEAR_SANDBOX_MAX_FILES")?.unwrap_or(3000);
```

subtle change, entirely up to you